tests/expectations/metrics/test_map_metric.py

```diff
@@ -201,11 +221,27 @@ def _expecation_configuration_to_validation_result_sql(
             execution_engine=execution_engine,
             assets={
                 "my_asset": {
-                    "table_name": "animal_names",
+                    "table_name": "animals_table",
                 },
             },
         )
     )
+
+    context.datasources["my_test_datasource"] = Datasource(
+        name="my_test_datasource",
+        execution_engine=execution_engine.config,
+        data_connectors={
+            "my_sql_data_connector": {
+                "class_name": "ConfiguredAssetSqlDataConnector",
+                "assets": {
+                    "my_asset": {
+                        "table_name": "animals_table",
```

Comment:
I believe this should be `animal_names` too

Reply:
@Shinnnyshinshin All fixed!  Thanks!